laws/ord.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const {lte, equals} = require('..');
+
+/**
+   ### Ord
+
+   1. `a.lte(b)` or `b.lte(a)` (totality)
+   2. If `a.lte(b)` and `b.lte(a)`, then `a.equals(b)` (antisymmetry)
+   3. If `a.lte(b)` and `b.lte(c)`, then `a.lte(c)` (transitivity)
+
+**/
+
+const totality = eq => f => g => {
+  const a = f[lte](g);
+  const b = g[lte](f);
+  const c = true;
+  return eq(a || b, c);
+};
```

Comment:
Given that I'm working on fantasyland/fantasy-laws#1, I'd very much like to know the reason. If it occurs to you, let me know. :)

Let's leave this as it is, Gabe. These functions will be removed from this repository in due course.